R/board_url.R

```diff
@@ -11,10 +11,19 @@
 #'
 #' `board_url()` is read only and does not currently support versions.
 #'
-#' @param urls A named character vector of URLs If the URL ends in a `/`,
-#'   `board_url` will look for a `data.txt` that provides metadata. The
-#'   easiest way to generate this file is to upload a pin directory created by
-#'   [board_folder()].
+#' @param urls identify available pins, possible values:
+#'   - unnamed character scalar, i.e. single string: URL to a manifest file.
+#'     If the URL ends in a `/`, `board_url()` will look for a `pins.txt`
+#'     containing the manifest. If manifest-file parses to a named list,
+#'     versioning is supported. If it parses to a named character vector,
+#'     the board will not support versioning.
+#'   - named list, values are character vectors of URLs, each element of the
+#'     vector refers to a version of the particular pin. If a URL ends in a `/`,
+#'     `board_url()` will look for a `data.txt` that provides metadata.
+#'   - named character vector of URLs: If the URL ends in a `/`,
+#'     `board_url()` will look for a `data.txt` that provides metadata. The
+#'     easiest way to generate this file is to upload a pin directory created by
+#'     [board_folder()]. Versioning is not supported.
```

Comment:
This is a lot for this section of the documentation. Can we have something shorter here, and then a bulleted list in **Details** that is more in depth?

Reply:
will do!